README.md

```diff
@@ -52,6 +52,7 @@
 │   │   ├── Hash Table Quadratic Probing.txt
 │   │   ├── HashTableLinearProbingC++.txt
 │   │   └── TwoSum.c++
+│   ├── Heuristic_Search.cpp
```

Comment:
You do not need to add this in README.md or project tree structure
That will be done automatically.